dataplane/dpdk.cpp

```diff
@@ -0,0 +1,28 @@
+#include <dpdk.h>
+
+namespace dpdk
+{
+
+std::optional<std::string> GetNameByPort(tPortId pid)
+{
+	char cname[256];
+	if (int res = rte_eth_dev_get_name_by_port(pid, cname); res)
```

Comment:
```suggestion
	std::array<char, 256> cname;
	if (int res = rte_eth_dev_get_name_by_port(pid, cname.data()); res)
```
Wow, found new github feature, didn't know about that